django/core/management/utils.py

```diff
@@ -122,3 +123,32 @@ def get_command_line_option(argv, option):
         return None
     else:
         return options.value
+
+
+def normalize_patterns(patterns):
+    """
+    Normalize an iterable of glob style patterns based on OS.
+    """
+    patterns = [os.path.normcase(p) for p in patterns]
+    dir_suffixes = {'%s*' % path_sep for path_sep in {'/', os.sep}}
+    norm_patterns = []
+    for p in patterns:
+        for dir_suffix in dir_suffixes:
+            if p.endswith(dir_suffix):
+                norm_patterns.append(p[:-len(dir_suffix)])
+                break
+        else:
+            norm_patterns.append(p)
+    return norm_patterns
+
+
+def is_ignored(path, ignore_patterns):
```

Comment:
This would benefit from a more detailed name since it's declared in a quite generic `utils` module.

What about `is_path_ignored`?